R-package/NAMESPACE

```diff
@@ -62,6 +64,7 @@ export(xgb.plot.tree)
 export(xgb.save)
 export(xgb.save.raw)
 export(xgb.set.config)
+export(xgb.slice.Booster)
```

Comment:
Is this still needed after having the s3 method?

Reply:
It's not needed, but I think it helps with discoverability compared to having just an S3 method.

Reply:
Thank you for sharing, sounds good!